### PR TITLE
New sol

### PR DESCRIPTION
**Pull request by aturkewi**
Update solution

@pletcher @mendelB 

Waiting to update all object collaboration at once. Feel free to comment.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -19,20 +19,30 @@ We recommend starting out by getting some of the initial `Artist` tests passing
...
 
-###`MP3Importer` class
-Let's start with the MP3 Importer. Build an `MP3Importer` class that parses a directory of files and sends the filenames to a song class to create a library of music with artists that are unique. To do this, you'll need two methods: `Mp3Importer#files` and `MP3Importer#import`. Your `MP3Importer` class should also have a `path` attribute that gets set on initialization.
+###`Song` class
 
-You should write code that responds to `MP3Importer.new('./db/mp3s').import`. Google around for how to get a list of files in a directory! Make sure you only get `.mp3` files.
+Let's start with the song class. 
 
-Since we have to send the filenames to the `Song` class, we'll end up calling the following code in the `#import` method: `Song.new_by_filename(some_filename)`. This will send us to the `Song` class, specifically `Song.new_by_filename`.
+Note: As you're working through the tests, you may notice that you'll need the Artist class to be working to some extent to pass all the Song class tests. You'll need to jump around and build up both class at the same time.
```
Comment (pletcher) [inline README.md]: "both class" -> "both classes"

File: lib/artist.rb [hunk 2]
```diff
@@ -1,20 +1,23 @@
...
   end
-
-  def add_song(song)
-    @songs << song
+  
+  def songs
+    Song.find_by_artist(self)
   end
 
+  # def add_song(song)
```
Comment (pletcher) [inline lib/artist.rb]: Why comment this out?

**Comment by pletcher** [comment]
This looks solid! I left just a couple minor comments.

File: lib/artist.rb [hunk 3]
```diff
@@ -1,20 +1,23 @@
 class Artist
   @@all = []
-  attr_accessor :name, :songs
+  attr_accessor :name#, :songs
```
Comment (mendelB) [inline lib/artist.rb]: I think you meant to delete the `:songs` accessor here?

**Comment by bhollan** [comment]
I helped a student today who was confused by some wording that would be changed by this PR.  I think this should be merged in.

**Comment by aturkewi** [comment]
I'm not sure we want to merge this one in just yet. 

I'm gonna ping @cernanb on this one.

This was part of a re-write of this section where we were going to train students to do OO associations differently. Instead of an Artist have a `@songs` instance variable with an array of songs, the Artist would have an instance method like `Song.all.select{ |s| s.artist == self}`. 

Mendel and I were part way through fixing this section up, but I think we need to merge these all in at once otherwise the curriculum will be teaching two different things (too confusing I think)

**Comment by drakeltheryuujin** [comment]
Looks to be part of the other out-of-date PR that I just closed. Closing this one as well. Not sure how relevant it is over a year later.

**Pull request closed without merging**